frontend/src/components/Item/WatchedIndicator.vue

```diff
@@ -1,17 +1,9 @@
 <template>
-  <v-chip color="success" class="card-chip" small>
+  <v-chip color="success" class="card-chip" size="small" variant="elevated">
     <v-icon>
       <i-mdi-check />
     </v-icon>
   </v-chip>
 </template>
 
 <script setup lang="ts" />
-
-<style lang="scss" scoped>
-.card-chip {
-  position: absolute;
-  top: 1em;
-  right: 1em;
-}
```

Comment:
Are we sure this class is needed both in Card component and here? I think it should only be necessary here (but one never knows with CSS!), so we don't need to decouple it at all (leaving it here only)

Reply:
Well the Card.vue file got this:

```vue
<v-progress-circular
            v-if="refreshProgress !== undefined"
            class="card-chip"
            rotate="-90"
            :model-value="refreshProgress"
            :indeterminate="refreshProgress === 0"
            color="white"
            size="24" />
```
and I checked without it, and I can assure you the class is needed lol